content/en/blog/2020/istiocon-2021/istiocon-2021.md

```diff
@@ -0,0 +1,16 @@
+# Join us for the first IstioCon in 2021!
+
+IstioCon 2021 will be the inaugural conference for Istio, the industry's most popular service mesh. In its inaugural year, IstioCon will be 100% virtual, connecting community members across the globe with Istio's ecosystem. This conference will take place at the end of February. 
+
+![/macruzbar/istio.io/content/en/blog/2020/istioconlogo.jpg]
+
+All the information related to IstioCon will be published on the [coference website](https://events.istio.io/). At this time, we encourage Istio users, developers, partners and advocates to [submit a session proposal through the conference's CFP](https://sessionize.com/istiocon-2021/). IstioCon will be an opportunity to showcase the lessons learned from running Istio in production, hands-on experiences from the Istio community, and featuring maintainers from across the Istio ecosystem. The conference offers a mix of keynotes, technical talks, lightning talks, workshops and roadmap sessions. Choose from the following formats to submit a session for IstioCon:
```

Comment:
change to "partners, and" (serial comma)

Reply:
added!